app/src/main/python/gogdl/dl/managers/v1.py

```
@@ -0,0 +1,313 @@
+        repository = dl_utils.get_zlib_encoded(self.api_handler, build['repository_manifest'])[0] or {}
+
+        size_data = self.manifest.calculate_download_size()
+
+        for depot in repository["depots"]:
+            if depot["dependencyId"] in self.manifest.dependencies_ids:
+                if not depot["executable"]["path"].startswith("__redist"):
+                    size_data[self.game_id]['*']["download_size"] += depot["compressedSize"]
+                    size_data[self.game_id]['*']["disk_size"] += depot["size"]
+
+        available_branches = set([build["branch"] for build in self.builds["items"] if build["branch"]])
+        available_branches_list = [None] + list(available_branches)
```

Ruff (0.12.2)

Consider `[None, *list(available_branches)]` instead of concatenation

Replace with `[None, *list(available_branches)]`

(RUF005)